CMS-Enterprise/mint-app
# [MINT-2550] Data Exchange Approach Marked Complete Notification

# MINT-2550

## Description


- Added notification for when Data Exchange Approach is marked complete

## How to test this change

1. Open Postman
2. Create a model plan
3. Set the Post Man user account notification preference flags to include both EMAIL and IN_APP (I do this directly in the database)
4. Update the Data Exchange Approach section
5. Upon setting the Marked Complete bool to true, you should see an email in Mailcatcher and a should be able to see an in-app notification by getting the user's notifications in postman

## PR Author Checklist




## PR Reviewer Guidelines

- It's best to pull the branch locally and test it, rather than just looking at the code online!
- When approving a PR, provide a reason _why_ you're approving it
  - e.g. "Approving because I tested it locally and all functionality works as expected"
  - e.g. "Approving because the change is simple and matches the Figma design"
- Don't be afraid to leave comments or ask questions, especially if you don't understand why something was done! (This is often a great time to suggest code comments or documentation updates)
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
